index.js

```diff
@@ -544,10 +545,34 @@ alexa.app = function(name, endpoint) {
     return self.handler;
   };
 
-  // A utility method to bootstrap alexa endpoints into express automatically
-  this.express = function(express, path, enableDebug) {
-    var endpoint = (path || "/") + (self.endpoint || self.name);
-    express.post(endpoint, function(req, res) {
+  // A utility method to bootstrap alexa endpoints into an express router
+  //
+  // @param object express the express app to attach to
+  // @param router express router instance to attach to the express app
+  // @param bool certCheck when true, applies alexa certificate checking (default true)
+  // @param bool enableDebug when true, sets up the route to handle GET requests (default false)
+  // @return ? TODO: not sure what this returns
+  this.express = function(express, router, certCheck, enableDebug) {
```

Comment:
This could probably raise an error if the second parameter is not a router (eg. a string), so that someone upgrading can "see" it.